apps/sensenet/src/context/repository-provider.tsx

```diff
@@ -21,6 +23,8 @@ export function RepositoryProvider({ children }: { children: React.ReactNode })
   const globalClasses = useGlobalStyles()
   const history = useHistory()
   const [authConfig, setAuthConfig] = useState<UserManagerSettings>()
+  const searchParams = new URLSearchParams(history.location.search)
+  const repoFromUrl = searchParams.get('repoUrl')
```

Comment:
```suggestion
  const repoFromUrl = useQuery().get('repoUrl')
```
Don't forget to import it from hooks.